descriptors/ModuleDescriptor-template.json

```diff
@@ -136,6 +136,21 @@
           "methods": ["POST"],
           "pathPattern": "/inn-reach/authentication",
           "permissionsRequired": ["inn-reach.authentication.item.post"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/inn-reach/central-servers/{centralServerId}/contribution-options",
```

Comment:
Please change to `/item-contribution-options` instead of `/contribution-options`
There are several contribution settings, simply saying `contribution-options` doesn't make it clear which one is referred to